### PR TITLE
Undo panda vis and merge fix spatial

### PR DESCRIPTION
**Pull request by Zbysekz**
This reverts pandaVis for htm_core nab detector & contains changes in #15 (there was merge conflicts)

### CODE REVIEW

**Comment by Zbysekz** [comment]
@breznak this seems good.

I will add thresholds and it should be ready

**Review by breznak — APPROVED** [review]
The diffs in HTMcore detector were a bit hard to read, but I tried to check throuroughly and seems good to me :+1: Thanks!

File: results/final_results.json [hunk 1]
```diff
@@ -15,9 +15,9 @@
         "standard": 16.43666922426724
     },
     "htmcore": {
-        "reward_low_FN_rate": 66.1922106936328,
-        "reward_low_FP_rate": 58.7930712907694,
-        "standard": 63.081419488725054
+        "reward_low_FN_rate": 76.5626293570994,
+        "reward_low_FP_rate": 61.359926511549155,
+        "standard": 71.3094612770284
```
Comment (breznak) [inline results/final_results.json:20]: this is with the "master" of htmcore? Are the results stable, ie reasonably consistent over several runs? If so, please also bump our score table in Readme and you can move us to the 1st place! ;)

Comment (Zbysekz) [reply results/final_results.json:20]: wait a second, i must push the non spatial numbers &try to build with latest htm.core

**Comment by breznak** [comment]
@Zbysekz I wanted to keep the conversations and linked PRs working, so I've done merged this PR into the original #15 . So closing here now, and we can continue against #15 and merge that to master when ready.

**Pull request closed without merging**